vendor/bat-native-ledger/src/bat/ledger/internal/media/twitch.cc

```diff
@@ -647,17 +614,22 @@ void MediaTwitch::SavePublisherInfo(const uint64_t duration,
                                     const ledger::VisitData& visit_data,
                                     const uint64_t window_id,
                                     const std::string& fav_icon,
-                                    const std::string& channel_id) {
-  if (channel_id.empty()) {
+                                    const std::string& channel_id,
+                                    const std::string& publisher_key) {
+  if (channel_id.empty() && publisher_key.empty()) {
     BLOG(ledger_, ledger::LogLevel::LOG_ERROR) <<
       "author id is missing for: " << media_key;
     return;
   }
 
-  std::string publisher_id = GetPublisherKey(channel_id);
-  std::string url = publisher_url + "/videos";
+  std::string key = publisher_key;
+  if (key.empty()) {
+    key = GetPublisherKey(channel_id);
+  }
 
-  if (publisher_id.empty()) {
+  const std::string url = publisher_url + "/videos";
```

Comment:
It seems like `url` isn't used anywhere, unless I'm just missing it.

Reply:
We're good on this:
https://github.com/brave/brave-core/pull/2618/files#diff-410ec7b37fbed59610d3617cce571df3R646

it's used outside of diff

Reply:
fixed